modules/ossm-running-v2-v3-cluster-wide-deployment-model.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-v2-v3-side-by-side-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-running-v2-v3-cluster-wide-deployment-model_{context}"]
+= Running {SMProductName} 2 and 3 using cluster wide deployment model
+//TP1 content influx. Title, etc may change.
+//No IA as of 10/23/2024 so this content is likely to move.
+
+If you are moving from {SMProductName} 2.6 in a cluster-wide deployment model, you can run {SMProduct} 2.6 side-by-side with {SMProduct} 3.0, in one cluster, without them interfering with each other.
+
+To prevent conflicts with {SMProduct} 3.0 when using the {SMProduct} 2.6 cluster-wide deployment model, you need to configure the `ServiceMeshControlPlane` resource to restrict namespaces to only those belonging to (SMProduct) 2.6.
+
+//NOTES: 10/23/2024
+
+//DO NOT USE VERSION NUMBER ATTRIBUTE. It is set for 3.0, will likely change, and it is unclear if later versions would impact this procedure for moving from 2.6.
+//Only moving from 2.6 has been tested and verified by QE.
+//Cluster-wide requires changes to OSSM 2.6 ServiceMeshControlPlane before installing OSSM 3.0
+
+.Prerequisites
+
+* You are running {ocp-product-title} 4.15 or later
```

Comment:
Why 4.15? OSSM 2.6 and 3.0 are both supported on OCP 4.14 onward. Is there a particular reason we indicate 4.15 here?

Reply:
@FilipB I believe you did the draft for this.. is that simply what you tested on, or is there a reason for only 4.15+?

Reply:
When I asked what OCP versions were supported, I was told 4.15 or later.